db/migrations/20190213160914_add_global_stat_setting_migration.php

```diff
@@ -21,6 +21,8 @@
  */
 
 use Phinx\Migration\AbstractMigration;
+use Jenssegers\Date\Date;
```

Comment:
I am not sure if phinx will be able to use this reference?

Reply:
Phinx isn't loaded like one of our normal endpoints, and I don't know if it loads the vendor files or not (I think perhaps not?)

You might need to use PHP core DateTime libs?